source/variables/colors.css

```diff
@@ -0,0 +1,14 @@
+/* 
+The color palette for the project.
+http://cssnext.io/features/#color-function 
+
+background-color: var(--primary);
+color: var(--greyDark);
+*/
+:root {
+	--primary: #e55030;
+	--secondary: #d1d2d4;
+	--grey-light: #eeeeee;
```

Comment:
Do all of these gray values need to be included? I know they're from Phoenix, but I've never understood there either why we’d want to _start all projects_ with three separate shades of gray. Or four, really, since `--secondary` is just another shade of gray. (And why are we using the British spelling of gray, while I'm at it? 😃 ) I feel that if these baseline variables are primarily meant to serve as examples more than anything else then we can get away with setting a single color variable here, maybe two, and calling it good.

```
--primary: tomato;
--secondary: bisque;
```